index.html

```diff
@@ -23,23 +38,24 @@ <h2>The Future</h2>
                 <h2>The Past</h2>
                 <p>Proin sed quam sed tellus vestibulum ultrices quis in nunc. Phasellus id dui id tortor tincidunt efficitur. Proin faucibus imperdiet erat, non varius lacus. Maecenas non nisl id turpis egestas tincidunt. Nam condimentum venenatis magna eget finibus.</p>
             </div>
+        </div>
         </section>
         
         <section class="middle-content">
         
             <h2>Why Did It Have To Be Boxes...</h2>
             
             <div class="boxes">
-                <div class="box">Box 1</div>
-                <div class="box">Box 2</div>
-                <div class="box">Box 3</div>
-                <div class="box">Box 4</div>
-                <div class="box">Box 5</div>
-                <div class="box">Box 6</div>
-                <div class="box">Box 7</div>
-                <div class="box">Box 8</div>
-                <div class="box">Box 9</div>
-                <div class="box">Box 10</div>
+                <div class="box" style = " background: teal; color: white;">Box 1</div>
+                <div class="box" style = " background: gold; color: white;">Box 2</div>
+                <div class="box" style = " background: cadetblue; color: white;">Box 3</div>
```

Comment:
Great job using inline styling to set the colors of the box, however, mixing inline styles with external styling libraries can get messy. Instead of storing your styles in one place, now you have 2.

A way to achieve the same result and make this more dynamic is to keep the "box" class and add an additional class to each box. For example:

```
<div class="boxes">
    <div class = "box box1">Box 1</div>
    <div class = "box box2">Box 2</div>
    <div class = "box box3">Box 3</div>
</div>
```

Reply:
You are right, that does make it less messy and more organized, Ill keep that tip in mind I forgot we can have multiple classes lol. Thanks again for feedback sir. :)